src/backend/common/half.hpp

```diff
@@ -832,9 +832,7 @@ class alignas(2) half {
 #endif
 
    public:
-#if CUDA_VERSION >= 10000
     AF_CONSTEXPR
```

Comment:
I believe this was added to circumvent another issue.

Reply:
I think the problem was the --device-as-default-execution-space flag for jit kernels. It was treating this function as a host only function in that case causing errors.